app/models/organisation.rb

```diff
@@ -8,9 +8,9 @@ class Organisation < ActiveRecord::Base
 
   has_many :permissions
   has_many :memberships
-  has_many :profiles, through: :memberships
+  has_many :profiles, -> { order :name }, through: :memberships
```

Comment:
Good call. Having both ```organisation.profiles``` and ```Profile.all``` returning results with the same ordering makes sense